README.md

```diff
@@ -7,38 +7,61 @@
 ## Introduction
 
 A highlevel interface for the dispatch API.
-The interface is made of the dispatch actor which should run once per microgrid.
-It provides two channels for clients:
-- "new_dispatches" for newly created dispatches
-- "ready_dispatches" for dispatches that are ready to be executed
 
-## Example Usage
+See [the documentation](https://frequenz-floss.github.io/frequenz-dispatch-python/v0.1/reference/frequenz/dispatch) for more information.
+
+## Usage
+
+The [`Dispatcher` class](https://frequenz-floss.github.io/frequenz-dispatch-python/v0.1/reference/frequenz/dispatch/#frequenz.dispatch.Dispatcher), the main entry point for the API, provides two channels:
+
+* [Lifecycle events](https://frequenz-floss.github.io/frequenz-dispatch-python/v0.1/reference/frequenz/dispatch/#frequenz.dispatch.Dispatcher.lifecycle_events): A channel that sends a message whenever a [Dispatch][frequenz.dispatch.Dispatch] is created, updated or deleted.
+* [Running status change](https://frequenz-floss.github.io/frequenz-dispatch-python/v0.1/reference/frequenz/dispatch/#frequenz.dispatch.Dispatcher.running_status_change): Sends a dispatch message whenever a dispatch is ready to be executed according to the schedule or the running status of the dispatch changed in a way that could potentially require the actor to start, stop or reconfigure itself.
```

Comment:
About these links, we were using redirect for version aliases because GitHub didn't support symlinks. With redirects we can't use direct links to an alias, but with symlinks we can. You could update the `alias_type` [configuration](https://github.com/jimporter/mike?tab=readme-ov-file#configuration) from `redirect` to `symlink` and then link to the `latest` alias for example, instead of `v0.1`.

That said, any linking to any particular version will be bad as things are now, because the README is also rendered as the home page of the project, and if you are visiting the docs for any particular version, the links in that version home page will point to a different version.

This is why I'm more inclined to **not** provide *deep links* to docs in the README, just explain in plain text and add a link to the docs home page (without any version specification, just https://frequenz-floss.github.io/frequenz-dispatch-python/) for people looking for more info.

That, or stop including the README entirely in the [`docs/index.md`](https://github.com/frequenz-floss/frequenz-dispatch-python/blob/v0.x.x/docs/index.md) and do some partial inclusions (and leave stuff with deep links to docs out of the included parts) like we do with channels:

* https://raw.githubusercontent.com/frequenz-floss/frequenz-channels-python/v1.x.x/README.md
* https://raw.githubusercontent.com/frequenz-floss/frequenz-channels-python/v1.x.x/docs/index.md

Happy to leave this for a future PR though.

Reply:
Let's do that in the next iteration that we already plan :) I am afraid to add more stuff here currently